extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -175,7 +175,10 @@ private function getTopContributorsDetails( Array $ids ) {
 	 * @return array
 	 */
 	private function getNavigationData() {
-		return $this->sendRequest( 'NavigationApi', 'getData' )->getData();
+		//getData always return array with navigation and wiki keys.
+		//See: includes/wikia/api/NavigationApiController.class.php:28
+		//and: includes/wikia/models/NavigationModel.class.php:175
+		return $this->sendRequest( 'NavigationApi', 'getData' )->getData()['navigation']['wiki'];
```

Comment:
This change is just a proposal.
It's related with https://github.com/Wikia/mercury/pull/1503